packet_testing/ext-hdr-test-packets.py

```diff
@@ -50,7 +23,7 @@ def run(self):
 
             # IP packet to be used for udp, tcp and icmp
             ip_packet = scapy.all.IP(src=self.__source_ip, dst=dest_ip)
-            ip_packet.options.append(IPOption_RFC3692_style_experiment(value=56059))
+            ip_packet.options.append(scapy.all.IPOption_RR(length=7, pointer=8, routers=['1.0.0.0']))
```

Comment:
Document that `pointer` is `1`-based, and that `length`s shorter than 39 seem to be handled correctly.